website/content/docs/k8s/upgrade/compatibility.mdx

```diff
@@ -15,7 +15,7 @@ the Helm chart which will ensure a compatible version of the Consul Kubernetes b
 
 | Consul Version | Compatible Consul Helm Versions |
 | -------------- | ------------------------------- |
-| 1.9.x          | 0.27.0 - 0.28.0                 |
+| 1.9.x          | 0.27.0 - 0.29.0                 |
```

Comment:
What about using `0.27.0-latest` and then we only have to update this when we bump consul to `1.10.x`?

Reply:
Good idea.